include/vinecopulib/vinecop/implementation/fit_controls.ipp

```diff
@@ -59,9 +61,13 @@ inline FitControlsVinecop::FitControlsVinecop()
 //!     pair copulas within a tree; never uses more than the number
 //!     of concurrent threads supported by the implementation.
 //! @param mst_algorithm The algorithm for building the maximum spanning
-//!     tree (`"prim"` or `"kruskal"`) during the tree-wise structure selection.
+//!     tree (`"prim"`, `"kruskal"`, `"random"`) during the tree-wise 
```

Comment:
"random" is probably not the best name, who knows what we add in the future. How about "wilson"?